api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -204,4 +200,26 @@ public User apply(User user) {
       }
     }, user);
   }
+
+  public void logAdminUserAction(long targetUserId, String targetAction, String oldValue, String newValue) {
```

Comment:
If you say `Object targetAction` you can get the above usage.

Reply:
I think you mean `Object oldValue`?